README.md

```diff
@@ -849,33 +849,42 @@ In `src/api/api.cc` we have:
 static Local<ObjectTemplate> ObjectTemplateNew(
     i::Isolate* isolate, v8::Local<FunctionTemplate> constructor,
     bool do_not_cache) {
+  ...
   i::Handle<i::Struct> struct_obj = isolate->factory()->NewStruct(
       i::OBJECT_TEMPLATE_INFO_TYPE, i::AllocationType::kOld);
-  i::Handle<i::ObjectTemplateInfo> obj = i::Handle<i::ObjectTemplateInfo>::cast(struct_obj);
-  InitializeTemplate(obj, Consts::OBJECT_TEMPLATE);
-  int next_serial_number = 0;
-  if (!constructor.IsEmpty())
-    obj->set_constructor(*Utils::OpenHandle(*constructor));
-  obj->set_data(i::Smi::zero());
+  i::Handle<i::ObjectTemplateInfo> obj =
```

Comment:
Refer to https://github.com/v8/v8/blob/ee74e718339223345b42935e9f4cbc7be208e557/src/api/api.cc#L1575-L1595